interfaces/cython/cantera/reactor.pyx

```diff
@@ -315,6 +315,16 @@ cdef class Reactor(ReactorBase):
         self.reactor.getState(&y[0])
         return y
 
+    def set_advance_limit(self, name, limit):
+        """
+        Limit absolute change of component *name* during `ReactorNet.advance`.
+        (positive *limit* values are considered; negative values are ignored). Note
+        that limits are disabled by default (with individual values set to -1.).
```

Comment:
I think it would be more clear to describe the handling of negative values as "Negative values disable a previously-set advance limit for a solution component" rather than saying that they are ignored.

Reply:
done.